elements/rh-table/rh-table.ts

```diff
@@ -0,0 +1,168 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+
+import { Logger } from '@patternfly/pfe-core/controllers/logger.js';
+
+import styles from './rh-table.css';
+import { RequestSortEvent, RhSortButton } from './rh-sort-button.js';
+export * from './rh-sort-button.js';
```

Comment:
Not sure this export needs to be here.   IIRC that would already be exported at the package level from `"exports": { ./*": "./elements/*"}` in package.json

Reply:
Done! Thx ")